Makefile

```diff
@@ -54,11 +52,6 @@ set-production-azure-subscription:
 	$(eval AZURE_RESOURCE_PREFIX=s189p01)
 	$(eval AZURE_ENV_TAG=Prod)
 
-.PHONY: set-key-vault-names
```

Comment:
is used by print-application-key-vault-name and print-infrastructure-key-vault-name which are used in workflows. So needs to return the vault names.